Import/Esri/ArcGISRuntime/Toolkit/Controls/CppApi/ArcGISCompass.qml

```diff
@@ -15,6 +52,13 @@ Item {
         autoHide: autoHideCompass
     }
 
+    /*!
+      \qmlproperty bool autoHideCompass
```

Comment:
is this a mistake? It seems like duplicate doc. We probably don't need to doc height, width, opacity, etc that comes from inheriting from Item